onnxruntime/test/python/contrib_ops/onnx_test_gridsampler.py

```diff
@@ -0,0 +1,533 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
For ORT just the unit tests in gridsampler_test.cc should be enough... these node tests should be added to ONNX PR...

Reply:
ok, I also see a few other ops added the onnx test, wiil remove it.